site/src/grammar-input.html

```diff
@@ -184,6 +184,9 @@ <h1>New Productions</h1>
 TypeArguments :
     AngleBracketedTokens
 
+ApplicationTypeArguments :
+    `::<` TypeArgumentList `>`
```

Comment:
How does this differ from the existing cases of `` `::` TypeArguments `` elsewhere in the grammar? Should they all be updated to use this production instead?

Reply:
Also, I don't see `TypeArgumentList` in the grammar, just `TypeArguments`, which already includes the brackets.

Reply:
Oops, yeah I extracted this from a larger patch so this got confused. I also realized that this might be the wrong solution for this issue with `CoverCallExpressionAndAsyncArrowHead` in the grammar, because the `TypeParameters_opt` there has to cover the async arrow head case, which doesn't use the `::`.

I'll close this PR and open a different one.